Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyIntCtrlCommon.h

```diff
@@ -93,23 +93,13 @@ namespace AzToolsFramework
                 toolTipString += "\n";
             }
             toolTipString += "[";
-            if (propertyControl->minimum() <= aznumeric_cast<AZ::s64>(QtWidgetLimits<T>::Min()))
-            {
-                toolTipString += "-" + QObject::tr(PropertyQTConstant_InfinityString);
-            }
-            else
-            {
-                toolTipString += QString::number(propertyControl->minimum());
-            }
+
+            const QString minString = QLocale().toString(propertyControl->minimum());
+            const QString maxString = QLocale().toString(propertyControl->maximum());
+
+            toolTipString += minString;
             toolTipString += ", ";
-            if (propertyControl->maximum() >= aznumeric_cast<AZ::s64>(QtWidgetLimits<T>::Max()))
-            {
-                toolTipString += QObject::tr(PropertyQTConstant_InfinityString);
-            }
-            else
-            {
-                toolTipString += QString::number(propertyControl->maximum());
-            }
+            toolTipString += maxString;
             toolTipString += "]";
```

Comment:
I'm guessing this is faster than
```
toolTipString += QString("[%1,%2]").args(propertyControl->minimum(), propertyControl->maximum());
```

Reply:
I'm not sure about speed. Without using QLocale though, it will be formatted without commas and I think its definitely not as nice to read (for large numbers). 

I couldn't quite get the args method to compile though. I tried this instead successfully:

            toolTipString += QString("[%1,%2]").arg(propertyControl->minimum()).arg(propertyControl->maximum());

I don't really have any preference. Which way to would you prefer?

Reply:
This works well I think:

```
            const QString minString = QLocale().toString(propertyControl->minimum());
            const QString maxString = QLocale().toString(propertyControl->maximum());
            toolTipString += QString("[%1,%2]").arg(minString).arg(maxString);
```

Reply:
Works for me